scalafix-core/src/main/scala/scalafix/rewrite/ExplicitReturnTypes.scala

```diff
@@ -58,14 +85,39 @@ case class ExplicitReturnTypes(mirror: Mirror)
         typ <- defnType(defn)
       } yield ctx.addRight(replace, s": ${typ.treeSyntax}")
     }.to[Seq]
+
+    import ctx.config.explicitReturnTypes._
+
+    def matchesMemberVisibility(mods: Seq[Mod]): Boolean =
+      memberVisibility.contains(visibility(mods))
+
+    def matchesMemberKind(defn: Defn): Boolean =
+      kind(defn).exists(memberKind.contains)
+
+    def matchesSimpleDefinition(rhs: Term): Boolean =
+      rhs.is[Lit] && skipSimpleDefinitions
```

Comment:
At ScalaDays Copenhagen, I remember chatting with @olafurpg about binpacking in scalafmt, and he said that he had binpacking behaving differently for literals and non-literals. Afterwards, the special behavior for literals ended up being so useful that Olaf upgraded it from just literals to literals and short identifiers.

When I saw a notion of a "simple definition" in this pull request, I thought that maybe that scalafmt story and this pull request relate in any way. Could you comment on that? /cc @olafurpg